x/auth/ante/basic.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/cosmos/cosmos-sdk/types/tx/signing"
 	"github.com/cosmos/cosmos-sdk/x/auth/legacy/legacytx"
 	authsigning "github.com/cosmos/cosmos-sdk/x/auth/signing"
+	authz "github.com/cosmos/cosmos-sdk/x/authz/exported"
```

Comment:
should auth depend on authz like this? I thought we were trying to avoid direct module dependencies.